net_sink.go

```diff
@@ -89,8 +96,7 @@ func NewNetSink(opts ...SinkOption) FlushableSink {
 		// arbitrarily buffered
 		doFlush: make(chan chan struct{}, 8),
 
-		// CEV: default to the standard logger to match the legacy implementation.
-		log: logger.StandardLogger(),
+		log: &loggingSink{},
```

Comment:
How will this work if an error is logged with this default sink? Do we need to null check s.writer in logMessage() or is the default value suitable? Assuming it is, would the default mean nothing is logged unless we specify a logger using WithLogger above?